polling_stations/apps/file_uploads/management/commands/check_pending.py

```diff
@@ -0,0 +1,31 @@
+from datetime import timedelta
+from django.conf import settings
+from django.utils.timezone import now
+from file_uploads.models import Upload
+from django.core.mail import EmailMessage
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        from_time = now() - timedelta(minutes=5)
+        qs = Upload.objects.filter(timestamp__lte=from_time).filter(
```

Comment:
Could you add this to [the Upload manger](https://github.com/DemocracyClub/UK-Polling-Stations/blob/master/polling_stations/apps/file_uploads/models.py#L28) as a new method?

This is because it's better to have this query defined in one place (on the model), so we can reuse it in other parts of the project, should we want to (for example, if we decide to show a page with them all on)